### ISSUE TITLE
Failing CI checks because of warnings in doctests

### ISSUE THREAD

**Issue by lbianchi-lbl** [nawi-hub/proteuslib#203]
The warnings (or at least some of them) seem to be related to recent deprecations in Pyomo's `Set` API:

```log
Document: how_to_guides/how_to_setup_simple_chemistry
-----------------------------------------------------
**********************************************************************
File "how_to_guides/how_to_setup_simple_chemistry.rst", line 811, in default
Failed example:
    # Import specific pyomo objects
    from pyomo.environ import ConcreteModel

    # Import the core idaes objects for Flowsheets and types of balances
    from idaes.core import FlowsheetBlock

    # Import the idaes objects for Generic Properties and Reactions
    from idaes.generic_models.properties.core.generic.generic_property import GenericParameterBlock
    from idaes.generic_models.properties.core.generic.generic_reaction import GenericReactionParameterBlock

    # Import the idaes object for the EquilibriumReactor unit model
    from idaes.generic_models.unit_models.equilibrium_reactor import EquilibriumReactor

    # Create an instance of a pyomo model
    model = ConcreteModel()

    # Add an IDAES flowsheet to that model
/home/runner/work/proteuslib/proteuslib/proteuslib/tools/parameter_sweep.py:104: UserWarning: Could not import mpi4py from current environment (defaulting to serial).
  warnings.warn("Could not import mpi4py from current environment (defaulting to serial).")
    model.fs = FlowsheetBlock(default={"dynamic": False})

    # Add a thermo parameter block to that flowsheet
    #   Here, we are passing our 'thermo_config' dictionary we created earlier
    model.fs.thermo_params = GenericParameterBlock(default=thermo_config)

    # Add a reaction parameter block to that flowsheet
    #   Here, we are passing our thermo block created above as the property package
    #   and then giving our 'reaction_config' as the instructions for how the
    #   reactions will be constructed from the thermo package.
    model.fs.rxn_params = GenericReactionParameterBlock(
                default={"property_package": model.fs.thermo_params, **reaction_config})

    # Add an EquilibriumReactor object as the unit model
    #   Here, we pass both the thermo package and reaction package, as well
    #   as a number of other arguments to help define how this unit process
    #   will behave.
    model.fs.unit = EquilibriumReactor(default={
                "property_package": model.fs.thermo_params,
                "reaction_package": model.fs.rxn_params,
                "has_rate_reactions": False,
                "has_equilibrium_reactions": True,
                "has_heat_transfer": False,
                "has_heat_of_reaction": False,
                "has_pressure_change": False})

    # At this point, you can 'fix' your inlet/outlet state conditions,
    #     setup scaling factors, initialize the model, then solve the model
    #     just as you would with any other IDAES flowsheet
Expected nothing
Got:
    WARNING: DEPRECATED: Using __getitem__ to return a set value from its
        (ordered) position is deprecated.  Please use card()  (deprecated in 6.1,
        will be removed in 7.0) (called from /usr/share/miniconda/envs/proteuslib-
        dev/lib/python3.6/site-packages/idaes/generic_models/properties/core/state
        _definitions/FTPx.py:143)
    WARNING: DEPRECATED: Using __getitem__ to return a set value from its
        (ordered) position is deprecated.  Please use card()  (deprecated in 6.1,
        will be removed in 7.0) (called from /usr/share/miniconda/envs/proteuslib-
        dev/lib/python3.6/site-packages/idaes/generic_models/properties/core/state
        _definitions/FTPx.py:148)
    WARNING: DEPRECATED: Using __getitem__ to return a set value from its
        (ordered) position is deprecated.  Please use card()  (deprecated in 6.1,
        will be removed in 7.0) (called from /usr/share/miniconda/envs/proteuslib-
        dev/lib/python3.6/site-packages/idaes/generic_models/properties/core/state
        _definitions/FTPx.py:148)
    WARNING: DEPRECATED: Using __getitem__ to return a set value from its
        (ordered) position is deprecated.  Please use card()  (deprecated in 6.1,
        will be removed in 7.0) (called from /usr/share/miniconda/envs/proteuslib-
        dev/lib/python3.6/site-packages/idaes/generic_models/properties/core/state
        _definitions/FTPx.py:148)
    WARNING: DEPRECATED: Using __getitem__ to return a set value from its
        (ordered) position is deprecated.  Please use card()  (deprecated in 6.1,
        will be removed in 7.0) (called from /usr/share/miniconda/envs/proteuslib-
        dev/lib/python3.6/site-packages/idaes/generic_models/properties/core/state
        _definitions/FTPx.py:143)
    WARNING: DEPRECATED: Using __getitem__ to return a set value from its
        (ordered) position is deprecated.  Please use card()  (deprecated in 6.1,
        will be removed in 7.0) (called from /usr/share/miniconda/envs/proteuslib-
        dev/lib/python3.6/site-packages/idaes/generic_models/properties/core/state
        _definitions/FTPx.py:148)
    WARNING: DEPRECATED: Using __getitem__ to return a set value from its
        (ordered) position is deprecated.  Please use card()  (deprecated in 6.1,
        will be removed in 7.0) (called from /usr/share/miniconda/envs/proteuslib-
        dev/lib/python3.6/site-packages/idaes/generic_models/properties/core/state
        _definitions/FTPx.py:148)
    WARNING: DEPRECATED: Using __getitem__ to return a set value from its
        (ordered) position is deprecated.  Please use card()  (deprecated in 6.1,
        will be removed in 7.0) (called from /usr/share/miniconda/envs/proteuslib-
        dev/lib/python3.6/site-packages/idaes/generic_models/properties/core/state
        _definitions/FTPx.py:148)
**********************************************************************
1 items had failures:
   1 of   3 in default
3 tests in 1 items.
2 passed and 1 failed.
***Test Failed*** 1 failures.

Doctest summary
===============
   21 tests
    1 failure in tests
    0 failures in setup code
    0 failures in cleanup code
build finished with problems.
```

**Comment by lbianchi-lbl** [comment]
As #205 shows, merging in IDAES/idaes-pse#470 should address these issues without any changes needed on Proteuslib's side.

**Comment by lbianchi-lbl** [comment]
Addressed by #205.
